vault/changelog.release.2022-05-10.md

```diff
@@ -0,0 +1,121 @@
+
+**File Nesting** File Nesting is now stable in v1.67 of VSCode. With this change, VSCode now has (partial) support for Dendron's dot hierarchy!
+
+Add the following to your workspace settings to see it in action:
+```json
+"explorer.fileNesting.enabled": true,
+"explorer.fileNesting.patterns": {
+    "*": "${basename}.*.md",
+}
+```
+
+- More information in the [VSCode Release Notes](https://code.visualstudio.com/updates/v1_67)
+
+**VSCode Markdown Features:** We published a [[blog post|dendron://dendron.blog/blog.2022.05-05-vscode-markdown-edition]] looking at the latest markdown features in VSCode. Learn about the new ways you drag and drop files, rename links and more. 
+
+### Starboard and TIL Highlights
+> These are highlights from the [[Dendron Discord|dendron://dendron.dendron-site/community.discord.channels]] `#starboard` and `#today-i-learned` channels.
+
+- ⭐ `@dagriefaa#2892` shared a tip for minimzing visual clutter in your workspace: _"you can use `<details>` and `<summary>` tags to set up collapsible regions"_
+
+```html
+<details>
+  <summary> click to show </summary>
+  hidden stuff
+</details>
+```
+
+- 💡 `foureyedsoul#0796` shared a link to [an article on Math Accessibility](https://www.ryanpatrickrandall.com/notes/Accessibility/math-a11y): _"This isn't exactly what I learned today,  but today I did write this  note summing up what I've recently learned about combining MathML & MathJax to make math formatting more accessible online"_
+
+### Dendron Reading Series
+
+This week's entry in the [[Dendron Reading Series|dendron://dendron.dendron-site/community.events.reading-series]].
+
+![[dendron://dendron.dendron-site/community.events.reading-series.2022.05.10.md]]
+
+### Event Reminders
+
+- **New User Tuesdays:** Visit the [[New User Tuesdays page|dendron://dendron.dendron-site/community.events.new-user-tuesdays]] for notes from previous sessions.
+    - Next: [Tue, May 10, 8:30 AM PDT / 15:30 UTC](https://link.dendron.so/luma)
+- **Greenhouse Talks:** Visit the [[Greenhouse Talks page|dendron://dendron.dendron-site/community.events.greenhouse]] for notes from previous sessions.
+    - Subject: TBD
+    - Description: TBD
+    - Next: [Fri, May 27, 04:00 PM PDT / 23:00 UTC](https://link.dendron.so/luma)
+    - [Greenhouse Talk Recordings - YouTube Playlist](https://link.dendron.so/greenhouse)
+- **Office Hours:** Visit the [[Office Hours page|dendron://dendron.dendron-site/community.events.office-hours]] for notes from previous sessions.
+    - Next: [Wed, Jun 01, 09:00 AM PST / 17:00 UTC](https://link.dendron.so/luma)
+    - [Office Hour Recordings - YouTube Playlist](https://link.dendron.so/6yPa)
+
+
+### Thank You's
+
+A big **thanks** to the following gardeners that brought up issues, contributions, and fixes to this release :man_farmer: :woman_farmer: 
+Visit [[Discord Roles|dendron://dendron.dendron-site/community.discord.roles]] for more information.
+
+- [Steven Hilton](https://github.com/mshiltonj) `@pengurrito#8324`
```

Comment:
i think @hikchoi added more contributions in [[Community|dendron://private/weekly.journal.2022.05.10.contributions#community]]